spec/template.dd

```diff
@@ -1082,6 +1082,19 @@ $(H2 $(LNAME2 function-templates, Function Templates))
         }
         ---
 
+    $(P Variadic Function templates can have parameters with default values.
+        These parameters are always set to their default value in case of IFTI.
+    )
+
+        ---
+        void $(CODE_HIGHLIGHT fun)(T...)(T t, string file = __FILE__)
```

Comment:
No. We have automatic code highlighting. Also please wrap this in:

```
$(SPEC_RUNNABLE_EXAMPLE_COMPILE
---
...
---
)
```

so that the spec tester can find it.

Reply:
done